docs/site/Routes.md

```diff
@@ -226,3 +226,62 @@ export class MyApplication extends RestApplication {
   }
 }
 ```
+
+## Mounting an Express Router
+
+If you have an existing [Express](https://expressjs.com/) application that you
+want to use with LoopBack 4, you can mount the Express application on top of a
+LoopBack 4 application. This way you can mix and match both frameworks, while
+using LoopBack as the host. You can also do the opposite and use Express as the
+host by mounting LoopBack 4 REST API on an Express application. See
+[Creating an Express Application with LoopBack REST API](express-with-lb4-rest-tutorial.md)
+for the tutorial.
+
+Mounting an Express router on a LoopBack 4 application can be done using the
+`mountExpressRouter` function provided by both `RestApplication` and
+`RestServer`.
```

Comment:
How about replacing naked class names with links to API docs?

- http://apidocs.loopback.io/@loopback%2fdocs/rest.html#RestApplication
- http://apidocs.loopback.io/@loopback%2fdocs/rest.html#RestServer